config/src/main/java/org/hyperledger/besu/config/experimental/ExperimentalEIPs.java

```diff
@@ -24,21 +24,9 @@
   // To make it easier for tests to reset the value to default
   public static final long EIP1559_BASEFEE_DEFAULT_VALUE = 1000000000L;
 
-  @Option(
-      hidden = true,
-      names = {"--Xeip1559-basefee-max-change-denominator"},
-      arity = "1")
-  public static Long basefeeMaxChangeDenominator = 8L;
-
   @Option(
       hidden = true,
       names = {"--Xeip1559-initial-base-fee"},
       arity = "1")
   public static Long initialBasefee = EIP1559_BASEFEE_DEFAULT_VALUE;
```

Comment:
Do we still have a use case for this cli option?  it seems awfully niche and is otherwise configurable via genesis file